ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -137,9 +137,10 @@ aws|aws-arm64)
     export KUBE_SSH_USER=core
     ;;
 azure4|azure-arm64) export TEST_PROVIDER=azure;;
-azurestack)
+azurestack|azurestack-qe)
```

Comment:
The cluster type for cluster profile `azurestack-qe` is `azurestack`, see code https://github.com/openshift/ci-tools/blob/master/pkg/api/types.go#L1372-L1375, so I think there is no need to add `azurestack-qe` here.